starlite/plugins/base.py

```diff
@@ -102,7 +137,7 @@ def is_plugin_supported_type(value: Any) -> TypeGuard[ModelT]:
         """
         raise NotImplementedError()
 
-    def to_dict(self, model_instance: ModelT) -> Union[Dict[str, Any], Awaitable[Dict[str, Any]]]:
+    def to_dict(self, model_instance: ModelT) -> "dict[str, Any]":
```

Comment:
maybe roll this change back given the headaches with testing using the blocking portal has caused in tortoise plugin.